### PR TITLE
[14.0][MIG] stock_move_change_source_location

### PR DESCRIPTION
**Pull request by AlvaroTForgeFlow**
Migration from v13 to v14 of change source location.

### CODE REVIEW

File: stock_move_change_source_location/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_stock_move_change_source_location_wizard,access_stock_move_change_source_location_wizard,model_stock_move_change_source_location_wizard,base.group_user,1,0,0,0
```
Comment (LoisRForgeFlow) [inline stock_move_change_source_location/security/ir.model.access.csv]: Could you change this to stock.group_stock_user,1,1,1,0

**Review by LoisRForgeFlow — COMMENTED** [review]
LG, could you squash your migration commits into a single one (the last 2 starting with 14.0 MIG)

File: stock_move_change_source_location/security/ir.model.access.csv (hunk 1 above)
Comment (AlvaroTForgeFlow) [reply stock_move_change_source_location/security/ir.model.access.csv]: Done, thanks!

File: stock_move_change_source_location/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_stock_move_change_source_location_wizard,access_stock_move_change_source_location_wizard,model_stock_move_change_source_location_wizard,base.group_user,1,1,1,0
```
Comment (MiquelRForgeFlow) [inline stock_move_change_source_location/security/ir.model.access.csv]:
```suggestion
access_stock_move_change_source_location_wizard,access_stock_move_change_source_location_wizard,model_stock_move_change_source_location_wizard,stock.group_stock_user,1,1,1,0
```

Comment (AlvaroTForgeFlow) [reply stock_move_change_source_location/security/ir.model.access.csv]: Sorry, now it should be ok

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Pull request merged**